build/Dockerfile-setup

```diff
@@ -38,7 +38,7 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     update-locale LANG=en_US.UTF-8
 
 ENV CASSANDRA_VERSION 3.11.13
-ENV CASSANDRA_URL https://downloads.apache.org/cassandra/$CASSANDRA_VERSION/apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz
+ENV CASSANDRA_URL https://archive.apache.org/dist/cassandra/$CASSANDRA_VERSION/apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz
```

Comment:
I chose to use the archived version of Cassandra rather than updating to the current. I expect this resource to be available for longer than the current download which will require less frequent updating.